elb_log_ingestor/main.py

```diff
@@ -66,26 +68,28 @@ def start_server():
         logs_to_be_processed, logs_processed, records, parser_stats
     )
     shipper = elasticsearch_shipper.ElasticsearchShipper(
-        #es_client, records, index_pattern, shipper_stats
-        None, records, index_pattern, shipper_stats
+        es_client, records, index_pattern, shipper_stats
```

Comment:
this was commented out before for local testing. It's now back to what it was before last week